net/znc/Makefile

```diff
@@ -31,6 +31,7 @@ define Package/znc/default
   CATEGORY:=Network
   TITLE:=ZNC
   URL:=http://en.znc.in/
+  USERID:=znc=65532:znc=65532
```

Comment:
Maybe use 6667 for user/group? (the IRC default port). That would match what some other daemons chose for their ids.